test/mocks/MarketPlace.sol

```diff
@@ -3,16 +3,29 @@
 pragma solidity 0.8.13;
 
 contract MarketPlace {
-    address[9] private marketsReturn;
+    struct MarketsCalledArgs {
+        uint256 maturity;
+        uint8 precision;
+    }
+
+    address private marketsReturn;
+    address private zcTokenMarketReturn;
 
-    mapping (address => uint256) public marketsCalled;
+    mapping (address => MarketsCalledArgs) public marketsCalled;
 
-    function marketsReturns(address[9] calldata m) external {
+    function marketsReturns(address m) external {
         marketsReturn = m;
     }
 
-    function markets(address u, uint256 m) external returns (address[9] memory) {
-        marketsCalled[u] = m;
+    function marketZcTokenReturns(address z) external {
```

Comment:
there's a method named `marketZcToken` ? wut.

Reply:
this is just a thing on the mock then? so you can manipulate getting the ZcToken?

this name would then dictate that the variable to return would be `marketZcTokenReturn`

Reply:
This is supposed to be a convenience method because a given `lend` or `redeem` call can interact with the protocols principal token and Illuminate's zc token for a given underlying and maturity. When the testing, if the 0 principal is passed, I want the mocked zc token to be returned, and otherwise we can use the protocol's return value.

Reply:
Renaming to `principalTokenReturns` and `zcTokenReturns` for clarity.